### ISSUE TITLE
RangeSlider stutters when the track is dragged in vertical orientation. 

### ISSUE THREAD

**Issue by macster110** [controlsfx/controlsfx#1099]
When the RangeSlider _track_ is dragged there is a visible stutter with both the low and high thumbs rapidly moving to other points on the track and then back again. The distance between the low and high thumb can also change. The effect is reduced by dragging the track very slowly. This does not occur when the RangeSlider is horizontal. Tested on the version 8.14.

**Comment by HGuillemet** [comment]
I can confirm this, with 11.0.0-SNAPSHOT. For instance when dragging the track upwards, some change events are received with a decreasing value.

**Comment by xupwup** [comment]
I also have this issue. I've written an ugly workaround for this which resolves the issue. It would be nice if this was fixed properly.
```
import impl.org.controlsfx.behavior.RangeSliderBehavior;
import impl.org.controlsfx.skin.RangeSliderSkin;
import javafx.application.Platform;
import javafx.scene.layout.StackPane;
import org.controlsfx.control.RangeSlider;

import java.lang.reflect.Field;

/**
 * Fixes https://github.com/controlsfx/controlsfx/issues/1099
 */
public class RangeSliderFix {
	@SuppressWarnings("unchecked")
	private static <T> T get(Object o, String v) throws IllegalAccessException, NoSuchFieldException {
		Field field = o.getClass().getDeclaredField(v);
		field.setAccessible(true);
		return (T) field.get(o);
	}

	static void workAroundRangeSliderBug(RangeSlider rangeSlider) {
		Platform.runLater(() -> {
			try {
				RangeSliderSkin skin = (RangeSliderSkin) rangeSlider.skinProperty().get();
				StackPane rangeBar = get(skin, "rangeBar");
				RangeSliderBehavior rangeSliderBehavior = get(skin, "rangeSliderBehavior");

				double[] preDragPos = new double[1];
				rangeBar.setOnMousePressed(e -> {
					rangeBar.requestFocus();
					preDragPos[0] = -e.getSceneY();
				});

				rangeBar.setOnMouseDragged(e -> {
					double delta = -e.getSceneY() - preDragPos[0];
					rangeSliderBehavior.moveRange(delta);
					preDragPos[0] = -e.getSceneY();
				});
			} catch (NoSuchFieldException | IllegalAccessException e) {
				e.printStackTrace();
			}
		});
	}
}

```

**Comment by james-d** [comment]
I inadvertently duplicated this with #1521. I have a proposed fix with PR #1522.
